apps/cms/src/lib/utils.ts

```
@@ -1,32 +1,131 @@
+  return twMerge(clsx(inputs));
 }
 
-export const getAppwriteImageUrl = (fileId: string, type: 'view' | 'preview' | 'download' = 'view', options?: {
+/**
+ * Normalize text by removing diacritics (accent marks) from characters
+ * This is useful for searching Vietnamese text without requiring exact diacritic matches
+ * @param text - The text to normalize
+ * @returns The normalized text with diacritics removed
+ */
+export function normalizeText(text: string): string {
+  return text.normalize("NFD").replace(/[\u0300-\u036f]/g, "");
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)